tests/acceptance/subscribers-test.js

```diff
@@ -198,6 +198,7 @@ describe('Acceptance: Subscribers', function() {
 
             // click the import subscribers button
             await click('.gh-btn:contains("Import CSV")');
+            expect(find('.gh-btn:contains("Import CSV")').hasClass('gh-btn-hover-green')).to.be.true;
```

Comment:
If you could. I think we only need to keep the addition of the class in `subscribers.hbs`